server/main.py

```diff
@@ -245,7 +229,38 @@ def add_text_to_image(image_path, text_from_prompt, file_number):
     except Exception as e:
         raise Exception(f"Error occurred during text addition: {e}")
 
+# Configure gemini api
+    
+def gemini_pro_api(prompt):
+
+    GEMINI_API_KEY = os.environ['GEMINI_API_KEY'] # API key for Gemini
+    genai.configure(api_key=GEMINI_API_KEY) # Configure the API key
+    model = genai.GenerativeModel('gemini-pro') # Load the model
+    response = model.generate_content(prompt) # Generate the response
+    return response.text
+
+# Configure chat gpt api
+
+def chat_gpt_api(prompt):
```

Comment:
```suggestion
def request_chat_gpt_api(prompt):
```
Function names should be an action verb conveying what kind of action you are taking.